test/groovy/TransportRequestUploadFileTest.groovy

```diff
@@ -57,6 +59,12 @@ public class TransportRequestUploadFileTest extends BasePiperTest {
                                          ]
                                      ]
                                  ]
+        Utils.metaClass.echo = { def m -> }
+    }
+
+    @After
+    public void tearDown() {
+        Utils.metaClass = null
```

Comment:
This change is also part of this PR?

Reply:
Yes. There is [this](https://github.com/SAP/jenkins-library/pull/1839/files#diff-1ab5d92264f5f099c7b3b5be320666c8R122) change which basically ensures the exception is not swallowed silently.

All the other changes are for the tests. Now we call `echo` which is expected to be fine in the free wild with the groovy environment on Jenkins. But outside that environment here in the tests "HE" is not able to resolve the `echo` step. Hence we have to mock it via the `metaClass` for `com.sap.piper.Utils`. And as clean-up we have to unset the echo. This can be done by "resetting" the meta-class of `com.sap.piper.Utils`.